doc/README.md

```diff
@@ -33,7 +32,7 @@ generation step run
 > make users-guide-requirements
 ```
 
-in the root of the repository.
+either from the root of the cabal repository or from the `docs/` subdirectory.
```

Comment:
Does this still work from the root?  Looks like the goal was deleted in the root Makefile.

Reply:
Sorry, you are right. I need to forward the "users-guide-requirements"  target from the root to the doc subdir as well.

Reply:
thanks